### PR TITLE
website: fixes redirected links

### PR DESCRIPTION
**Pull request by BRKalow**
Updates links pointing at redirects to point at the redirect destination.

### CODE REVIEW

File: website/content/docs/operations/monitoring-nomad.mdx [hunk 1]
```diff
@@ -292,22 +292,22 @@ latency and packet loss for the [Serf] address.
...
+[metric-types]: /docs/operations/metrics#metric-types
 [metrics-api-endpoint]: /api-docs/metrics
 [prometheus-telem]: /docs/configuration/telemetry#prometheus
 [`plan_rejection_tracker`]: /docs/configuration/server#plan_rejection_tracker
 [serf]: /docs/configuration#serf-1
 [statsd-exporter]: https://github.com/prometheus/statsd_exporter
 [statsd-telem]: /docs/configuration/telemetry#statsd
 [statsite-telem]: /docs/configuration/telemetry#statsite
-[tagged-metrics]: /docs/telemetry/metrics#tagged-metrics
+[tagged-metrics]: /docs/operations/metrics#tagged-metrics
```
Comment (thiskevinwang) [inline website/content/docs/operations/monitoring-nomad.mdx:310]:
```suggestion
[tagged-metrics]: /docs/operations/metrics-reference#tagged-metrics
```

File: website/content/docs/operations/monitoring-nomad.mdx [hunk 2]
```diff
@@ -292,22 +292,22 @@ latency and packet loss for the [Serf] address.
...
-[allocation-metrics]: /docs/telemetry/metrics#allocation-metrics
+[allocation-metrics]: /docs/operations/metrics#allocation-metrics
 [circonus-telem]: /docs/configuration/telemetry#circonus
 [collection-interval]: /docs/configuration/telemetry#collection_interval
 [datadog-alerting]: https://www.datadoghq.com/blog/monitoring-101-alerting/
 [datadog-telem]: /docs/configuration/telemetry#datadog
 [draining]: https://learn.hashicorp.com/tutorials/nomad/node-drain
 [gh-11830]: https://github.com/hashicorp/nomad/pull/11830
-[metric-types]: /docs/telemetry/metrics#metric-types
+[metric-types]: /docs/operations/metrics#metric-types
```
Comment (thiskevinwang) [inline website/content/docs/operations/monitoring-nomad.mdx:302]:
```suggestion
[metric-types]: /docs/operations/metrics-reference#metric-types
```

File: website/content/docs/operations/monitoring-nomad.mdx [hunk 3]
```diff
@@ -292,22 +292,22 @@ latency and packet loss for the [Serf] address.
 
 [alerting-rules]: https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/
 [alertmanager]: https://prometheus.io/docs/alerting/alertmanager/
-[allocation-metrics]: /docs/telemetry/metrics#allocation-metrics
+[allocation-metrics]: /docs/operations/metrics#allocation-metrics
```
Comment (thiskevinwang) [inline website/content/docs/operations/monitoring-nomad.mdx:295]:
```suggestion
[allocation-metrics]: /docs/operations/metrics-reference#allocation-metrics
```

**Review by thiskevinwang — APPROVED** [review]
Left 3 fixes. LGTM otherwise!

**Comment by BRKalow** [comment]
@thiskevinwang should this redirect be updated? 🤔

**Comment by BRKalow** [comment]
@thiskevinwang Oh interesting, so these were redirect chains 😱

File: website/content/docs/runtime/interpolation.mdx [hunk 4]
```diff
@@ -63,227 +63,37 @@ Below is a full listing of node attributes that are interpretable. These
...
-      <td>
-        <code>{'${attr.os.version}'}</code>
-      </td>
-      <td>Version of the client OS</td>
-    </tr>
-  </tbody>
-</table>
+| Property                                               | Description                                                                                                                                            |
+| ------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `{'${attr.cpu.arch}'}`                                 | CPU architecture of the client (e.g. `amd64`, `386`)                                                                                              |
```
Comment (zchsh) [inline website/content/docs/runtime/interpolation.mdx:80]: In the preview, I'm seeing these as brackets render:

### [Upstream (HTML syntax)](https://www.nomadproject.io/docs/runtime/interpolation)

CleanShot 2022-10-18 at 10 05 56@2x

### [Preview from this PR](https://nomad-hvhp9clzl-hashicorp.vercel.app/docs/runtime/interpolation)

CleanShot 2022-10-18 at 10 05 46@2x

Can we do something like this:

```suggestion
| `${attr.cpu.arch}`                                 | CPU architecture of the client (e.g. `amd64`, `386`)                                                                                              |
```

... or does that fail due to MDX or something?

Comment (BRKalow) [reply website/content/docs/runtime/interpolation.mdx:80]: Good catch, I think it should be a-ok to remove these, one moment...

**Review by zchsh — APPROVED** [review]
Ah whoa nice catch with the `<thing>` portions, I didn't even see those 😬 💯

**Pull request merged**